src/plugin/morphologyHDF5.h

```diff
@@ -25,7 +25,7 @@ class MorphologyHDF5
     bool _readV11Metadata();
     bool _readV2Metadata();
     HighFive::DataSet _getStructureDataSet(size_t nSections);
-    void _readPoints(unsigned int);
```

Comment:
Should we add a unit test illustrating the difference between `unsigned int` and `int` in the signature?

Reply:
I would like to skip that bit. I just fixed the interface to reflect the internal logic.

Reply:
I mean, there should have been a unit test exercising  `firstSectionOffset == -1` branch, that should have caught type mismatch.